pkg/ebpf/c/tracee.bpf.c

```diff
@@ -6264,16 +6270,31 @@ int sched_process_exit_signal(struct bpf_raw_tracepoint_args *ctx)
     if (unlikely(signal == NULL))
         return 0;
 
-    struct task_struct *task = (struct task_struct *) bpf_get_current_task();
+    // Hashes
 
     u64 id = bpf_get_current_pid_tgid();
     u32 host_tid = id;
     u32 host_pid = id >> 32;
-    u64 start_time = get_task_start_time(task);
+
+    struct task_struct *task = (struct task_struct *) bpf_get_current_task();
+    if (task == NULL)
+        return -1;
+
+    struct task_struct *leader = get_leader_task(task);
+    struct task_struct *parent = get_leader_task(get_parent_task(leader));
 
     // The hash is always calculated with "task_struct->pid + start_time".
-    u32 hash = hash_task_id(host_tid, start_time);
-    save_to_submit_buf(&signal->args_buf, (void *) &hash, sizeof(u32), 0);
+    u32 task_hash = hash_task_id(get_task_host_tgid(task), get_task_start_time(task));
```

Comment:
ditto - shouldn't it be the task_pid for the task?

Reply:
got rid of bpf_get_current_pid_tgid() call (no need).